homeassistant/scripts/check_config.py

```diff
@@ -22,8 +22,6 @@
 from homeassistant.exceptions import HomeAssistantError
 
 REQUIREMENTS = ('colorlog==4.0.2',)
-if system() == 'Windows':  # Ensure colorama installed for colorlog on Windows
```

Comment:
Is this no longer necessary?

Reply:
It’s automatic these days. In the past you needed `colorlog[windows]`

See https://pypi.org/project/colorlog/ search windows/colorama